rfcs/026-relevance_reporting_service/README.md

```diff
@@ -0,0 +1,123 @@
+# RFC 026: Relevance reporting service
+
+**Last updated: 06 July 2020.**
+
+## Glossary
+
+* **elastic-query:** A blob of JSON we send to Elasticsearch
+* **search-terms:** A set of terms sent as a string from a client to a search service
+
+## Summary
+
+A way for us to be able to test and report on the efficacy of elastic-queries by being able to compare a set of search-terms and their respective expected results and ordering.
+
+## Motivation
+
+We need to have and instil transparency and confidence in the way that our search relevancy works.
+
+Any test to help with this will need to be against our production data, as that is exactly what people will be seeing.
+
+### Use cases
+
+* Developers and data-scientists being able to write new search queries with  confidence in improvement and without fear of regression
+* Allowing internal teams to have a greater understanding and more input as to how the search works, thus being able to share this with external researchers
+* External researchers interested in how their results are being returned
+
+## Proposal
+
+![architecture](./relevance_service.png)
```

Comment:
I find the architecture diagram & the explanation below quite confusing (I found I needed the context of the Elasticsearch rank eval docs to understand better).

Some explanation of the terms used and some examples of what they are ("query", "elastic-query", "search terms") would be useful for me.

Reply:
There is a glossary at the top of the doc outlining those queries - does that need to be made clearer?

Reply:
For me, yes: "A blob of JSON we send to Elasticsearch" doesn't tell me what it is for.

Here's my understanding: 

1. The service accepts a query with some expected results. 
2. The service compares the results of that query on real data to the expected results. 
3. The services responds with an evaluation of how well the expected results match reality. 

The service is a layer on top of the rank-eval API in Elasticsearch allowing you to record interactions and take actions based on responses.